tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -210,197 +182,112 @@ def teardown_method(self):
-                                                               hostname=hostname)
-            assert_utils.assert_true(resp[0], resp[1])
-            server_data_pod_list.remove(pod_name)
-            LOGGER.info("Step 5: Services of deleted server pod %s are in offline state", pod_name)
-
-            LOGGER.info("Step 6: Check services status on remaining pods %s", server_data_pod_list)
-            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
-                                                               fail=False)
-            LOGGER.debug("Response: %s", resp)
-            assert_utils.assert_true(resp[0], resp)
-            LOGGER.info("Step 6: Services of remaining pods are in online state")
-
-            LOGGER.info("Step 7: Perform READs and verify DI on the written data")
+            self.restore_method = const.RESTORE_DEPLOYMENT_K8S
+            self.restore_pod = True
+            assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+            LOGGER.info("Deleted %s server pod %s by deleting deployment (unsafe)", count,
+                        pod_name)
+            LOGGER.info("Step 3: Perform READs and verify DI on the written data")
             resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                         log_prefix=self.test_prefix,
                                                         skipwrite=True, skipcleanup=True,
-                                                        nsamples=2, nclients=2)
+                                                        nsamples=2, nclients=2,
+                                                        setup_s3bench=False)
             assert_utils.assert_true(resp[0], resp[1])
-            LOGGER.info("Step 7: Performed READs and verified DI on the written data")
-
-        LOGGER.info("%s (K) server pods shutdown one by one successfully and read/verify "
-                    "after each pod down verified", self.kvalue)
-
+            LOGGER.info("Step 3: Performed READs and verified DI on the written data")
+        LOGGER.info("Step 2: %s (K) server pods shutdown one by one successfully, Verified "
+                    "cluster, services states are as expected & remaining pods status is online"
+                    " and read/verify after each pod down verified", self.kvalue)
         LOGGER.info("Completed: Test to verify degraded READs after each pod failure till K "
                     "server pods down - unsafe shutdown.")
 
     # pylint: disable=too-many-locals
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-40573")
-    @CTFailOn(error_handler)
-    def test_deletes_kserver_pods_down(self):
+    def test_deletes_kserver_pods_fail(self):
         """
-        This test tests DELETEs after all K server pods down - unsafe shutdown
+        This test tests Degraded DELETEs after all K server pods down - unsafe shutdown
```

Comment:
These are not actually degraded deletes, performance is degrading but cluster doesn't go to degraded state after server pod down. Should we remove "degraded"? @sonalk0209